cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -109,9 +112,9 @@ ExternalProject_Add(
     "@HUNTER_PACKAGE_INSTALL_PREFIX@"
         # not used, just avoid creating Install/<name> empty directory
     UPDATE_COMMAND
-    "@CMAKE_COMMAND@" -P
-    "@HUNTER_GLOBAL_SCRIPT_DIR@/append-boost-config-macros.cmake"
-    "@HUNTER_Boost_CMAKE_ARGS@"
+    "@CMAKE_COMMAND@"
+    "-DBOOST_CONFIG_MACROS=${boost_config_macros_list}"
```

Comment:
`examples/Boost-custom-args` is not working.

This line converted to `-DBOOST_CONFIG_MACROS=BOOST_1` `BOOST_2` `BOOST_3=5` (3 arguments, 2 ignored) instead of `-DBOOST_CONFIG_MACROS=BOOST_1;BOOST_2;BOOST_3=5` (1 argument). It's better to use `configure_file` to create script.